config/api.yml

```diff
@@ -268,7 +268,7 @@
       :post:
       - :name: edit
         :identifier: host_edit
-        :disabled: true
+        :disabled: false
```

Comment:
Missed this,  just remove the :disabled line here, also do the same the same on the collection side so multiple hosts can be targeted with a single post. Thanks.